src/installer/managed/Microsoft.NET.HostModel/ComHost/ClsidMap.cs

```diff
@@ -65,7 +67,7 @@ public static void Create(MetadataReader metadataReader, string clsidMapPath)
 
             using (StreamWriter writer = File.CreateText(clsidMapPath))
             {
-                writer.Write(JsonSerializer.Serialize(clsidMap, new JsonSerializerOptions { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull }));
+                writer.Write(JsonSerializer.Serialize(clsidMap, s_jsonOptions));
```

Comment:
Wouldn't it be better to serialize directly to the `FileStream`?

Reply:
It probably is but those optimizations are beyond the scope of this PR and this isn't product code so the performance gains probably are probably not that significant.

Reply:
It would be nice if there was an analyzer for this as well.